E2E/e2eTests/Source/Abilities/Sdk.swift

```diff
@@ -0,0 +1,221 @@
+import Foundation
+import PrismAgent
+import Builders
+import Combine
+import Domain
+import Pluto
+import Core
+
+class Sdk: Ability {
+    typealias T = Client
```

Comment:
Because one is the ability class for the actor and the other is the ability itself

```swift
        let presentation = try await cloudAgent.using(
            ability: OpenEnterpriseAPI.self,
            action: "ask a presentation proof to \(connectionId)"
        ).requestPresentProof(connectionId)
```